.github/workflows/bigtable-emulator-system-tests.yaml

```diff
@@ -26,7 +26,7 @@ jobs:
       - name: Setup PHP
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.1'
```

Comment:
incrementing emulators to 8.1 since we'll be dropping php 8.0 in May so we might as well do it now